ci-operator/step-registry/openshift-pipelines/install-and-tests/README.md

```diff
@@ -0,0 +1,39 @@
+# openshift-pipelines-install-and-tests-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+- [Prerequisite(s)](#prerequisite--s-)
+  - [Infrastructure](#infrastructure)
+  - [Environment Variables](#environment-variables)
+- [Custom Images](#custom-images)
+
+## Purpose
+
+To execute Openshift Pipelines interop tests. All XML results will be saved into "$ARTIFACT_DIR".
+
+## Process
+
+This script does the following to run Openshift Pipelines interop tests
+1. Login to the test cluster as a kubeadmin.
+2. Run gauge command for olm.spec to install openshift pipelines operator.
+3. Run other gauge specs for interop tests.
+4. Move xmls to `${ARTIFACT_DIR}/junit_*.xml`.
+
+## Prerequisite(s)
+
+### Infrastructure
+
+- A provisioned test cluster to target.
+ 
+### Environment Variables
```

Comment:
Do you want to add other env variables used in the commands.sh

Reply:
I think those vars which can be updated from config file are added in this `Environment Variables` section. 
Variables that are added in the commands.sh should not be changed. Those are set to store test artifacts.